hugo-modules/core/utils/get-params.html

```diff
@@ -14,7 +14,7 @@
 {{ $content_type := .content_type }}
 {{ $params := dict }}
 
-{{ if (eq $content_type "page") }}
+{{ if (or (eq $content_type "page") (hasPrefix $content_type "t-")) }}
```

Comment:
Hi @bezoerb, there is still a breaking change here that we need to resolve. If the content type is `t-default` (legacy), then it tries to get the `.params` directly, as it would be correct for `t-page`. But they are in `data` and would have to be fetched via `utils/get-data`. With the `hasPrefix $content_type "t-"` we take `t-page` and `t-default` in the same way, which was fine in other places, but not here.

Reply:
I have completely removed the check on page type. Instead I try to get page params directly and if they are empty I use get-data.